src/OpenTelemetry.Instrumentation.ConfluentKafka/ConfluentKafkaCommon.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+
+namespace OpenTelemetry.Instrumentation.ConfluentKafka;
+
+internal static class ConfluentKafkaCommon
+{
+    internal static readonly string ActivitySourceName = typeof(ConfluentKafkaInstrumentation).Assembly.GetName().Name!;
+
+    internal static readonly ActivitySource ActivitySource = new(ActivitySourceName);
```

Comment:
Pass version argument, it can be calculated based on `Assembly.GetPackageVersion()` see AssemblyVersionExtensions class.

Reply:
implemented through `ConfluentKafkaCommon.InstrumentationVersion`